src/utils/macros.h

```diff
@@ -225,5 +226,13 @@ extern const char* g_death_file;
 /// Macro to signal a function that the result must be used.
 #define MUST_USE_RESULT __attribute__((__warn_unused_result__))
 
+#ifdef ESP32
```

Comment:
Could we handle this through openmrn_features.h

Reply:
Perhaps, this is more of a temp fix until esp-idf updates for Balazs' PR. Perhaps remove the esp32 check and instead check for definition of the method only instead?

Reply:
This is a specific workaround for a specific bug in the ESP32 SDK. I am not aware of this bug existing anywhere else.